src/main/net/polydawn/mdm/commands/MdmStatusCommand.java

```diff
@@ -52,6 +57,17 @@ public MdmExitMessage call() throws IOException {
 			assertInRepo();
 		} catch (MdmExitMessage e) { return e; }
 
+		if (depName != null) {
```

Comment:
On further thought: this change is probably fine as-is; invoking the `MdmModuleSet` path will be slower on large projects since it will construct a full working-tree status on every module, which simply isn't necessary for this task.